sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -267,12 +283,23 @@ public static boolean hasCrashedInLastSession() {
 
     /**
      * Provides information about any available crash report from the last session, if it crashed.
+     * This method is a synchronous call and blocks UI thread.
```

Comment:
Javadoc: not UI thread but calling thread. If you call it in background it won't block UI thread.